trucks.js

```diff
@@ -509,3 +509,44 @@ var foodTrucks = [
 // getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
 // filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
 // filterByFoodType(foodType) - return trucks with associated 'foodType'
+var _ = require('underscore')
+
+var getTrucks = function getTrucks(){
+	return foodTrucks;
+};
+
+var getTruck = function getTruck(name){
+	var rightTruck = _.find(foodTrucks, function(name){
+		return foodTrucks.name === name;
```

Comment:
When you're looping over the foodTrucks array with _.find, the value that gets passed to the second parameter, the anonymous function, is a foodTruck object (you've called it 'name'.) In the body of that function you want to compare the name of the foodTruck, name.name in your case, to name. What would probably make the code more clear would be to name that parameter truck instead. The result of the current code is that nothing is returned, as you're currently checking against a parameter, name, on foodTrucks, which doesn't have a property called 'name'.